dist/obs-server.spec

```diff
@@ -338,6 +366,26 @@ export DESTDIR=$RPM_BUILD_ROOT
   perl -p -i -e 's/^APACHE_GROUP=.*/APACHE_GROUP=apache/' Makefile.include
 %endif
 
+# run gem clean up script
+/usr/lib/rpm/gem_build_cleanup.sh %{buildroot}/%_libdir/obs-api/ruby/*/
+# remove cached gems
+rm -rf src/api/vendor/cache/*
+# Remove extensions, we don't need them
```

Comment:
Hm. Really? I very much doubt that.

Reply:
This is what you need:
`/usr/lib64/ruby/gems/2.5.0/extensions/x86_64-linux/2.5.0/nokogiri-1.8.4/nokogiri/nokogiri.so`

This is what he's removing:
```
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/.sitearchdir.-.nokogiri.time
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/depend
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/extconf.rb
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/html_document.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/html_document.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/html_element_description.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/html_element_description.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/html_entity_lookup.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/html_entity_lookup.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/html_sax_parser_context.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/html_sax_parser_context.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/html_sax_push_parser.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/html_sax_push_parser.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/nokogiri.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/nokogiri.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/nokogiri.so
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_attr.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_attr.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_attribute_decl.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_attribute_decl.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_cdata.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_cdata.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_comment.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_comment.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_document.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_document.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_document_fragment.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_document_fragment.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_dtd.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_dtd.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_element_content.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_element_content.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_element_decl.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_element_decl.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_encoding_handler.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_encoding_handler.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_entity_decl.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_entity_decl.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_entity_reference.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_entity_reference.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_io.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_io.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_libxml2_hacks.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_libxml2_hacks.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_namespace.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_namespace.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_node.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_node.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_node_set.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_node_set.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_processing_instruction.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_processing_instruction.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_reader.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_reader.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_relax_ng.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_relax_ng.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_sax_parser.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_sax_parser.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_sax_parser_context.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_sax_parser_context.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_sax_push_parser.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_sax_push_parser.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_schema.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_schema.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_syntax_error.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_syntax_error.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_text.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_text.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_xpath_context.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xml_xpath_context.h
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xslt_stylesheet.c
/usr/lib64/ruby/gems/2.5.0/gems/nokogiri-1.8.4/ext/nokogiri/xslt_stylesheet.h
```

The comment is just misleading

Reply:
Ah they get installed to `lib` okay...